### ISSUE TITLE
training performance on pre trained models

### ISSUE THREAD

**Issue by soans1994** [susantabiswas/facial-keypoint-regression#1]
hello author,

i tried similar keypoint detection with pre trained(imagenet)  vgg and mobilenet models. but the training accuracy is 1% , why is that. when i try simple model with few conv blocks, the accuracy is upto 40%. am i doing something wrong? my dataset is hand dataset with hand keypoints.

edit: i used vgg16 pre trained model as follows:

from keras.layers import Input, Lambda, Dense, Flatten
from keras.models import Model, Sequential
from keras.applications.vgg16 import VGG16, preprocess_input
import tensorflow as tf

input_shape = (224,224,3)
num_classes = 42
def model(input_shape,num_classes):

    vgg = VGG16(
    include_top=False, weights="imagenet", input_tensor=None,
    input_shape=input_shape)
    """
    for layer in vgg.layers:
        layer.trainable=False
    x = Flatten()(vgg.output)
    output = Dense(num_classes, activation="softmax")(x)
    model = Model(vgg.input, output)
    """
    vgg.trainable=True
    global_average_layer = tf.keras.layers.GlobalAveragePooling2D()
    prediction_layer = tf.keras.layers.Dense(num_classes, activation='softmax')
    model = tf.keras.Sequential([ vgg, global_average_layer, prediction_layer])
    model.summary()
    return model
if __name__=="__main__":
    model(input_shape,num_classes)

thank you
